framework/source/class/qx/test/Mixin.js

```diff
@@ -240,8 +240,8 @@ qx.Class.define("qx.test.Mixin",
           {
             var s = "";
             if (!dontRecurs) {
-              this.__b = new qx.Patch2();
-              s += "++" + this.__b.foo(true) + "__";
+              this['__b'] = new qx.Patch2();
```

Comment:
If you declare __b as member variable then the patch mechanism ```qx.Class.include(qx.Patch1, qx.MPatch)``` complains about __b is not allowed to be overridden.